test/DynamoCoreUITests/RecordedTests.cs

```diff
@@ -33,6 +34,10 @@ public class RecordedTests : DSEvaluationViewModelUnitTest
         private IEnumerable<string> customNodesToBeLoaded = null;
         private CommandCallback commandCallback = null;
 
+        // Geometry preloading related members.
+        protected bool preloadGeometry;
+        protected Preloader preloader;
+
```

Comment:
Things get more interesting with `RecordedTests` -- by default the test setup does not preload ASM. As it turns out, few tests failed because they need ASM preloaded. So a new `bool RecordedTests.preloadGeometry` member is added for whichever test that needs ASM preloaded. It is as easy as setting `preloadGeometry = true` before calling `RunCommandsFromFile`.